.travis.yml

```diff
@@ -37,6 +37,8 @@ script:
   - catkin build --limit-status-rate 0.1 --no-notify
   # Run tests
   - catkin run_tests
+  # check test (this only works on indigo)
```

Comment:
i mean you need to use rostest version 0.6.15 to property catch the error on the test, hydro and indigo/release did not, only indigo/shadow-fixed provide this latest rostest
% this is super complex things, so you better not dig into this. https://github.com/jsk-enshu/robot-programming/blob/master/.travis.yml#L22